plugins/search-backend-module-elasticsearch/api-report.md

```diff
@@ -421,7 +421,7 @@ export interface OpenSearchElasticSearchClientOptions
   // (undocumented)
   nodes?: string | string[] | OpenSearchNodeOptions | OpenSearchNodeOptions[];
   // (undocumented)
-  provider?: 'aws';
+  provider?: 'aws' | 'opensearch';
```

Comment:
Could you please update the documentation here?
https://backstage.io/docs/features/search/search-engines#example-configurations